src/sidebar/templates/annotation.html

```diff
@@ -34,15 +33,15 @@
       ng-click="vm.toggleCollapseBody($event)"
       ng-title="vm.collapseBody ? 'Show the full annotation text' : 'Show the first few lines only'"
       ng-bind="vm.collapseBody ? 'More' : 'Less'"
-      h-branding="accentColor">mire</a>
+      h-branding="accentColor">more</a>
```

Comment:
Noticed this while editing the template and, while I'm not sure what the full picture here is, I'm fairly confident that this shouldn't be _mire_

Reply:
That looks like a classic Kyle typo

Reply:
good catch